frappe/core/doctype/communication/communication.py

```diff
@@ -463,7 +463,10 @@ def update_first_response_time(parent, communication):
 			first_responded_on = communication.creation
 			if parent.meta.has_field("first_responded_on") and communication.sent_or_received == "Sent":
 				parent.db_set("first_responded_on", first_responded_on)
-			parent.db_set("first_response_time", round(time_diff_in_seconds(first_responded_on, parent.creation), 2))
+			if parent.doctype != "Issue":
```

Comment:
Is there no better way to achieve what you're trying to do? There's no `Issue` DocType in Frappe.

Reply:
Took your initial suggestion and used Communication's `on_update` hook instead, so I'll close this PR.